api/global/internal/meter_test.go

```diff
@@ -225,3 +225,23 @@ func TestDefaultSDK(t *testing.T) {
 	require.Equal(t, `{"updates":[{"name":"test.builtin{A=B}","sum":1}]}
 `, <-ch)
 }
+
+func TestUnbindThenRecordOne(t *testing.T) {
+	internal.ResetForTest()
+
+	// Note: this test uses oppsite Float64/Int64 number kinds
```

Comment:
Delete this comment.

Reply:
Oops, copy pasta. Will fix in a separate PR.